frontend/src/components/layout/SignUp.jsx

```diff
@@ -0,0 +1,31 @@
+
+import styles from './SignUp.module.css'
+
+const SignUp = () =>{
+    return (
+        <>
+        <div className = {styles.container}>
+                <section className = {styles.Headings}>
+                     <h1  className = {styles.header}>SignUp</h1>
+                     <p id = {styles.Name}>Start learning the fundamaentals of Coding</p>\
```

Comment:
There is a typo in "fundamentals".
I believe that className should be in lower case. You have className={styles.header}, but className={styles.Headings}. The same with id.

Reply:
As a convention, we should make all our class names camelCase (starting with a lowercase).